data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
+		Sunbeam 4
```

Comment:
This is a 1/3 drop in firepower, which is a huge loss.

Reply:
I'm not following, this beetle was outfittes to beat humans this time, not hai or Korath

Reply:
There are three Korath vessels in that fight too. The Sunbeam Shield Beetle was designed to beat anything, which is what I'd expect of this General. You have a good point though, that he would have modified the design for this specific battle.

Reply:
Thunderhead would do some good work against those raiders as well
But yeah thats partly why I enhanced this ship (its got more shield and shield regen), the battle was too rough and korath weapons can heat you up a bunch